applications/Polkadot-Protocol-Conformance-Tests.md

```diff
@@ -0,0 +1,163 @@
+
+### Legal Structure
+
+- **Registered Address:** Bulgaria, Dragan Tsankov 23A, 1113, Sofia, Bulgaria
+- **Registered Legal Entity:** LimeLabs Ltd.
+
+### Team's experience
+
+At LimeChain, we possess considerable expertise in developing various tools, including [Gosemble, a framework for building Substrate compatible Runtimes in Go](https://github.com/LimeChain/gosemble), [Fruzhin, a Host implementation in Java](https://github.com/limechain/fruzhin), [a framework for runtimes in AssemblyScript](https://github.com/LimeChain/subsembly), a framework for runtimes in AssemblyScript. On top of that, we’re working on a [Parachain Validation Conformance Testing suite](https://polkadot.polkassembly.io/motion/389), have substantial experience in Rust/WebAssembly developer tooling from [Matchstick](https://github.com/limeChain/matchstick/) and actively contribute to infrastructure projects in Cosmos and Hedera Hashgraph.
+
+### Team Code Repos
+
+- [https://github.com/LimeChain](https://github.com/LimeChain)
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- [https://github.com/vikinatora](https://github.com/vikinatora)
+
+### Team LinkedIn Profiles (if available)
+
+- [https://www.linkedin.com/in/viktor-todorov-8a7434122/](https://www.linkedin.com/in/viktor-todorov-8a7434122/)
+- [https://www.linkedin.com/in/chris-veselinov/](https://www.linkedin.com/in/chris-veselinov/)
+
+## Development Status :open_book:
+
+No actual development has been made for this RFP. However, the team has spent a significant amount of time delving into the Host’s specification as part of the Java Host implementation we're currently working on. Additionally, our work on the PVF conformance testing suite has enhanced our understanding of how to address challenging-to-test sections of the code.
+
+## Development Roadmap :nut_and_bolt:
+
+The primary framework under consideration for this research will be Zombienet, as it already provides the groundwork for these types of tests. Each of the steps outlined below will be dedicated to investigating the creation of Zombienet tests for the specified functionalities. Depending on the feasibility of this, the team will either:
+
+- Develop a Proof of Concept (PoC) Zombienet test for the specified functionality.
+    - If certain edge cases cannot be tested using the current version of Zombienet, we will outline the necessary development steps to enable such testing.
+- Document in the research report the development steps required to enable testing of the specified functionality using Zombienet.
+
+Outlined below are the testing scenarios for each Host module that will be the focus of the research:
+
+1. Host API
+    1. Storage
+    2. Child Storage
+    3. Hashing
+    4. Trie
+2. SCALE
+    1. SCALE encoding
+    2. SCALE decoding
+3. State Trie
+    1. Trie encoding
+    2. Trie decoding
+    3. Trie proof generation
+4. BABE
```

Comment:
Can you elaborate more on this? Does block import include forking and longest chain related properties?

Reply:
We’ve intentionally left out specific properties for BABE and GRANDPA as our main goal will be to establish a way to test any BABE functionality/property which implicitly includes forking and longest chain related properties. If you have specific properties that you’d want us to test, feel free to note them here as we won’t be able to cover all of them in the specified timeline.